README.md

```diff
@@ -38,7 +38,7 @@ cd into the cloned folder and run npx hint. and npx stylelint "**/*.{css,scss}"
 ## Latest Issue
```

Comment:
### Optional
- [ ] It would be better if we improve our README file so that non-developers can also use the repo. For example; when we say `git clone the repo` we can make it better in this way `git clone https://github.com/KatIsCoding/Portfolio`. Also, it will be better to deploy the app so that everyone can have access to the app on the go